nf_core/download.py

```diff
@@ -282,6 +300,7 @@ def download_workflow_platform(self, location=None):
             remote_url=f"https://github.com/{self.pipeline}.git",
             revision=self.revision if self.revision else None,
             commit=self.wf_sha.values() if bool(self.wf_sha) else None,
+            additional_tags=self.additional_tags if self.additional_tags else None,
```

Comment:
```suggestion
            additional_tags=self.additional_tags,
```
self.additional_tags is already tuple/string/None